stk/calculators/optimization/macromodel.py

```diff
@@ -608,6 +615,46 @@ def __init__(self,
                          timeout=timeout,
                          use_cache=use_cache)
 
+    @staticmethod
+    def _check_params(
+        minimum_gradient,
+        maximum_iterations
+    ):
+        """
+        Check if the optimization parameters are valid for MacroModel.
+
+        MacroModel ``.com`` files have well-defined column widths and
+        special ways to implement too long input. This method raises
+        `MacroModelInputError` exception if the input cannot be
```

Comment:
https://code-style.readthedocs.io/en/latest/python.html#docstring-markup

But more importantly, the comment about what gets raised should be put into a Raises section at the end of the docstring.